frontend/lib/tracking/ga.ts

```diff
@@ -0,0 +1,109 @@
+import { getCookie } from '../cookie';
+
+interface TrackerConfig {
+    name: string;
+    id: string;
+    sampleRate: number;
+    siteSpeedSampleRate: number;
+}
+
+const getQueryParam = (key: string): string => {
```

Comment:
Should the return type of `getQueryParam` not be `string || undefined`? If there are no `params` or `pairs`, or `key` isn't present in `pairs` it'll return `undefined`.